runtime/buildtools.mk

```diff
@@ -110,7 +114,7 @@ NLS_NLSTOOL := $(JAVA) -cp sourcetools/lib/j9nls.jar com.ibm.oti.NLSTool.J9NLS
 nls : buildtools
 	$(NLS_NLSTOOL) $(NLS_OPTIONS)
 
-HOOK_DEFINITION_FILES := $(strip $(call findAllFiles,.,*.hdf))
+HOOK_DEFINITION_FILES := $(call convertUnixToWinPath,$(strip $(call findAllFiles,.,*.hdf)))
```

Comment:
This shouldn't be necessary (nor have any effect): all the files should be of the form `./subdir/xxx.hdf`.

Reply:
just close it

Reply:
removed